jujugui/static/gui/src/app/app.js

```diff
@@ -752,7 +755,7 @@ YUI.add('juju-gui', function(Y) {
           switchModel={views.utils.switchModel.bind(
             this, this.createSocketURL.bind(this), this.switchEnv.bind(this))}
           user={this._getAuth()}
-          users={this.get('users')}
+          users={Y.clone(this.get('users'), true)}
```

Comment:
Is there a way to do a clone here without YUI? We've been trying to keep YUI out of these components.